package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -550,18 +570,57 @@ def _read_next_timestep(self):
             alpha = beta = gamma = 90.
         ts.dimensions = xlen, ylen, zlen, alpha, beta, gamma
 
+        _get_column = lambda ids, col_ids : [col_ids[i] for i in ids if
```

Comment:
Gratuitous use of `lambda` ;-) — you could have also used
```python
def _get_column(...):
   return ...
```
Defining in-block functions is not frowned upon, to my knowledge.

Reply:
And it could even be a static method.